src/dot.rs

```diff
@@ -6,7 +6,7 @@ use serde::{Deserialize, Serialize};
 use crate::quickcheck::{Arbitrary, Gen};
 
 /// Dot is a version marker for a single actor
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize, Ord)]
```

Comment:
Why do you want to order Dots? In the general case we don't treat actor ids as comparable so for the case `Dot { actor: "a", counter: 2 }` and `Dot {actor: "b": counter: 2}` we can't order these two dots